Libs/MRML/Widgets/qMRMLThreeDViewControllerWidget.cxx

```diff
@@ -333,6 +333,15 @@ void qMRMLThreeDViewControllerWidget::updateWidgetFromMRML()
     return;
     }
 
+  QStringList AxesLabels;
```

Comment:
I guess that would be Slicer guide style .. both are similar: http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Style_Guide#Code_style